packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/tests/ReviewWorkflows.test.js

```diff
@@ -111,8 +111,10 @@ describe('Admin | Settings | Review Workflow | ReviewWorkflowsPage', () => {
     expect(queryByText('Workflow is loading')).not.toBeInTheDocument();
   });
 
-  test('display stages', () => {
-    const { getByText } = setup();
+  test('display stages', async () => {
+    const { getByText, queryByText } = setup();
+
+    await waitFor(() => expect(queryByText('Workflow is loading')).not.toBeInTheDocument());
```

Comment:
how come you have to `waitFor`? 🤔

Reply:
Let me check again if it is really needed. Overall it makes sense, because the whole page is not mocked and gets the data from msw, so there is a chance the test is flaky without.

Reply:
Oh I see, yes that makes more sense then because it'll behave more like the normal application – might be worth writing a generic comment at the top of the page to clarify this if you need to keep it :) only because this isn't our defacto way of testing w/data (yet)